src/containers/AutomaticPlatformOptimization/AutomaticPlatformOptimizationCard.js

```diff
@@ -247,6 +291,16 @@ function getDefaultHostnames(hostname, isSubdomain) {
   return defaultHostnames;
 }
 
+async function apoHeaderIsPresent() {
+  try {
+    let origin = new URL(document.URL).origin;
+    let res = await fetch(origin);
+    return res.headers.has('cf-edge-cache');
+  } catch (err) {
```

Comment:
could we log the error just in case?
```suggestion
  } catch (err) {
    console.warn(err);
```